src/EditorFeatures/Test/EmbeddedLanguages/StackFrame/StackFrameParserTests.cs

```diff
@@ -519,4 +518,18 @@ public void TestLanguages(string at, string @in, string line)
                 line: CreateToken(StackFrameKind.NumberToken, "16", leadingTrivia: [CreateTrivia(StackFrameKind.LineTrivia, $"{line} ")]),
                 inTrivia: CreateTrivia(StackFrameKind.InTrivia, $" {@in} "))
                 );
+
+    [Fact]
+    public void TestStateMachineMethod()
```

Comment:
What happens if there's a state machine for a local async method? Some code like:

```csharp
Task T()
{
    return Inner();

    async Task Inner()
    {
        await Task.Yield();
        throw new Exception();
    }
}
```

Gives me a name like <T>g__Inner|1_0.MoveNext()